x-pack/plugins/cases/public/common/test_utils.ts

```diff
@@ -0,0 +1,12 @@
+/*
```

Comment:
Do we use this file for cases? If not I think we should remove it.

Reply:
yes. the function `removeExternalLinkText` is used in `markdown_editor/renderer.test.ts`. PS - pull my branch and you can figure out if we need these files. the answer is YES!! i moved over files as needed